dist/classes/GoldRush.js

```diff
@@ -0,0 +1,69 @@
+class GoldRush extends Matrix {
+    constructor(rowNum, colNum) {
+        super(rowNum, colNum)
+        this.players = { 1: { row: 0, col: 0, points: 0 }, 2: { row: rowNum - 1, col: colNum - 1, points: 0 } }
+        this.directions = { up: { name: "row", val: -1 }, down: { name: "row", val: 1 }, left: { name: 'col', val: -1 }, right: { name: 'col', val: 1 } }
+        this.coinCuonter = 0
+        this.loadBoard = this.loadBoard()
+    }
+
+    loadBoard() {
+        this.loadPlayers()
+        this.generateItems("c")
+        this.generateItems("b")
+    }
+
+    loadPlayers() {
+        this.alter(this.players[1].row, this.players[1].col, "1")
+        this.alter(this.players[2].row, this.players[2].col, "2")
+    }
+
+    generateItems(val) {
+        let item
+        if (val === "c") {
+            item = Math.floor(Math.random() * (((this.rowNum * this.colNum) - 2) / 2)) + this.rowNum
```

Comment:
this line: `Math.floor(Math.random() * (((this.rowNum * this.colNum) - 2)` appears few time, you should extract it to a util function